test/cmd/get.sh

```diff
@@ -24,22 +24,15 @@ os::cmd::expect_success_and_text "_output/oshinko get" "abc"
 os::cmd::expect_success_and_text "_output/oshinko get" "def"
  
 # check for columns
-os::cmd::expect_success '_output/oshinko-cli get abc | grep -e "^abc\s*[012]\s*spark://abc:7077\s*http://abc-ui:8080\s*abc-ui-route.*\s*Running\s*<shared>$"'
+os::cmd::expect_success '_output/oshinko-cli get abc | grep -e "^abc\s*[012]\s*Running"'
 
 # incomplete clusters
 os::cmd::expect_success "oc delete service abc-ui"
 os::cmd::expect_success_and_text "_output/oshinko-cli get abc" "Incomplete"
-os::cmd::expect_success_and_text "_output/oshinko-cli get abc" "<missing>"
 
 # no such cluster
 os::cmd::expect_failure_and_text "_output/oshinko get nothere" "no such cluster 'nothere'"
 
-# verbose
```

Comment:
I wonder if the verbose flag for get should give us the old columns, for backwards compat? Just in case someone is consuming them